src/main/java/nl/tudelft/contextproject/model/PhysicsObject.java

```diff
@@ -12,8 +12,9 @@
 	 * Get spatial object with physics control.
 	 * Each Collidable object has to have some kind of spatial object that 
 	 * implements the PhysicsControl interface.
+	 *
 	 * @return
-	 * 		an object that implements PhysicsControl.
+	 * 		an object that implements PhysicsControl
 	 */
-	public PhysicsControl getPhysicsObject();
+	PhysicsControl getPhysicsObject();
```

Comment:
IntelliJ automatically hates you for using these modifiers, which is why I removed them.